frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -157,7 +157,12 @@ const PVCTableRow = connect(mapStateToProps)(({ obj, metrics }) => {
         )}
       </TableData>
       <TableData className={tableColumnClasses[7]}>
-        <ResourceKebab actions={menuActions} kind={kind} resource={obj} />
+        <ResourceKebab
+          actions={menuActions}
+          kind={kind}
+          resource={obj}
+          hoverMessage={t('public~Resource is being deleted.')}
```

Comment:
Is this the only reason the action could be disabled?

Reply:
This will only be disabled when the object is undergoing deletion: https://github.com/openshift/console/pull/11860#discussion_r949229659.